static/templates/settings/account-settings.handlebars

```diff
@@ -80,3 +83,8 @@
 
   </div>
 </div>
+<script>
+$(function () {
+    $('[data-toggle="tooltip"]').tooltip();
+});
+</script>
```

Comment:
Thanks for working on this @amalad! 

We don't put inline script tags in handlebars templates; please instead move the JS code to static/js/settings.js.  Also, if you `git grep tooltip`, you'll see that you should use a more precise selector for this sort of thing, like this:

`static/js/ui.js:    $('.message_failed i[data-toggle="tooltip"]').tooltip();`